tdms/tests/system/test_regen.py

```diff
@@ -77,6 +77,9 @@ def workflow(test_id: str, preserve_inputs: bool = PRESERVE_FLAG) -> None:
 
 # Run each system test that is currently saved in the repository, as identified through the config files
 # "MATLAB doesn't run on GH runners. If you want to check that regenerating the input data still allows the tests to pass, run this locally and remove the skip mark."
+@pytest.mark.filterwarnings(
+    "ignore::DeprecationWarning"
+)  # hdf5storage.savemat throws a comparison warning from within numpy
```

Comment:
Ah, right. The (much much) lesser of the evils.

Reply:
Yeah :/ _Although_ I can make a ticket to use `h5py` rather than `hdf5storage` to do the "adjusting". 

Now we know that only `-v7.3` `.mat` files are `HDF5` (and we're enforcing saving in this convention), we can in theory have the "adjust" method read/write via `h5py` assuming there's no funny business with `MATLAB` -> `Python` datatypes.